pytest_tests/steps/cluster_test_base.py

```diff
@@ -17,18 +19,24 @@ def fill_mandatory_dependencies(self, cluster: Cluster, client_shell: Shell):
         yield
 
     @allure.title("Tick {epochs_to_tick} epochs")
-    def tick_epochs(self, epochs_to_tick: int):
+    def tick_epochs(self, current_epoch: int, epochs_to_tick: int):
         for _ in range(epochs_to_tick):
             self.tick_epoch()
+            self.wait_for_epochs_align(current_epoch)
 
     def tick_epoch(self):
         epoch.tick_epoch(self.shell, self.cluster)
 
-    def wait_for_epochs_align(self):
-        epoch.wait_for_epochs_align(self.shell, self.cluster)
+    def wait_for_epochs_align(self, current_epoch_number: Optional[int] = None):
+        epoch.wait_for_epochs_align(self.shell, self.cluster, current_epoch_number)
 
     def get_epoch(self):
         return epoch.get_epoch(self.shell, self.cluster)
 
     def ensure_fresh_epoch(self):
         return epoch.ensure_fresh_epoch(self.shell, self.cluster)
+
+    @allure.step("Tick Epochs and wait for epochs align")
```

Comment:
`for epoch alignment` or `for epochs to align`

Reply:
Fixed